src/actions/catalogActions.js

```diff
@@ -6,10 +6,19 @@ import yaml from 'js-yaml';
 function loadCatalogIndex(catalog) {
   return fetch(catalog.spec.storage.URL + 'index.yaml', { mode: 'cors' })
     .catch(() => {
-      console.log(`Fetch error for ${catalog.spec.storage.URL}, attempting with cors anywhere.`);
-      return fetch('https://cors-anywhere.herokuapp.com/' + catalog.spec.storage.URL + 'index.yaml', { mode: 'cors' });
+      console.log(
+        `Fetch error for ${
```

Comment:
console log or console error?

Reply:
At this point, its not an error yet. We are handling an error by trying an alternative method.

Just wanted to give some sign that we are in this path, since it won't be clear otherwise.